View/Xaml2/Pages/Server/PluginsPage.xaml.cs

```diff
@@ -23,6 +29,148 @@ public PluginsPage(PluginViewModel pluginViewModel)
             DataContext = viewModel;
         }
 
+        private async void InstallLocal_PreviewMouseLeftButtonDown(object sender, System.Windows.Input.MouseButtonEventArgs e)
+        {
+            System.Windows.Forms.OpenFileDialog dialog = new System.Windows.Forms.OpenFileDialog
+            {
+                Filter = "Jar file|*.jar",
+                Multiselect = true,
+                InitialDirectory = System.Convert.ToString(
+                    Microsoft.Win32.Registry.GetValue(
```

Comment:
I suppose this is a hack to get to the downloads folder by default right?

Is there any other way which is possible in .NET without accassing a certain entry in the registry. I fear that this will break some day and that this is not doable in the recode of Fork where we also target other (non Windows) Platforms.
If it is not possible please wrap it in a try/catch with a Fallback Value for now

Reply:
> I suppose this is a hack to get to the downloads folder by default right?

Basically yes, it's a hack to obtain the downloads folder even if the user updated its location.
You are right, it might break but for now it works.

It might be good to add a default location if Directory.Exists fails for the downloads folder